source/ip/icmp.md

```
@@ -26,8 +26,9 @@
 - 2オクテットの『チェックサム』
 - 2オクテットの『識別子』(データ)
     - データ部分の長さや、種類に応じたデータや元パケットの情報などが適宜渡されます
-    - 例えばEchoメッセージ(タイプ8や0)では、識別子(2オクテット)とシーケンス番号(2オクテット)となります
-    - その他メッセージの種類に応じた情報や、エラー通知においては元のIPパケットのヘッダの一部などが格納されています
+        - 例えばEchoメッセージ(タイプ8や0)では、識別子(2オクテット)とシーケンス番号(2オクテット)となります
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 4; Actual: 8

(MD007, ul-indent)